nil/cmd/nild/main.go

```diff
@@ -148,6 +148,7 @@ func parseArgs() *nildconfig.Config {
 
 	// For backward compatibility
 	rootCmd.PersistentFlags().IntVar(&cfg.RPCPort, "port", cfg.RPCPort, "http port for rpc server")
+	rootCmd.PersistentFlags().IntVar(&cfg.PprofPort, "pprof-port", cfg.PprofPort, "port to serve pprof profiling information")
```

Comment:
You need to add default to `NewDefaultConfig`. Otherwise without option you will start pprof on 0 port

Reply:
Yep, you're right. Fixed.